common_version.txt

```diff
@@ -0,0 +1 @@
+dev
```

Comment:
what is this file used for? :) should it be dev?

Reply:
Removed, it got replaced with changelog.